neighsyncd/neighsync.cpp

```diff
@@ -141,6 +142,20 @@ bool NeighSync::isLinkLocalEnabled(const string &port)
 {
```

Comment:
Update this function description as it checks global Ipv6 LLA status as well now.

Reply:
Solved